site2/docs/reference-pulsar-admin.md

```diff
@@ -2369,20 +2348,279 @@ Remove the message TTL for a topic.
 
 Usage
 ```bash
-$ pulsar-admin topics remove-message-ttl tenant/namespace/topic
-=======
+$ pulsar-admin topics remove-message-ttl tenant/namespace/topic 
+```
+
+Options 
+|Flag|Description|Default|
 |---|---|---|
 |`--enable`, `-e`|Enable message deduplication on the specified topic.|false|
 |`--disable`, `-d`|Disable message deduplication on the specified topic.|false|
 
+### `get-deduplication`
+Get a deduplication policy for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics get-deduplication tenant/namespace/topic
+```
+
+### `set-deduplication`
+Set a deduplication policy for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-deduplication tenant/namespace/topic options
+```
+
 ### `remove-deduplication`
-Remove a deduplication policy from a topic.
+Remove a deduplication policy for a topic.
 
 Usage
 ```bash
 $ pulsar-admin topics remove-deduplication tenant/namespace/topic
 ```
 
+### `get-retention`
+Get the retention policy that is applied to a topic.
+
+Usage
+```bash
+$ pulsar-admin topics get-retention tenant/namespace/topic
+```
+
+### `set-retention`
+Set the retention policy that is applied to a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-retention tenant/namespace/topic options
```

Comment:
shall we describe the available options?

Reply:
There are no examples for the commands on the namespace level. So probably it's unnecessary.